unittest/unit/build.js

```diff
@@ -671,20 +696,63 @@ describe('build', () => {
       };
       const {ast} = flattenIDL(specIDLs, customIDLs);
       const interfaces = ast.filter((dfn) => dfn.type === 'interface');
-      const exposureSet = getExposureSet(interfaces[0]);
+      const exposureSet = getExposureSet(interfaces[0], scopes);
       assert.hasAllKeys(exposureSet, ['Worker']);
     });
+
+    it('"GlobalScope" stripped from exposure set names', () => {
+      const specIDLs = {
+        first: WebIDL2.parse(
+          `[Exposed=AudioWorkletGlobalScope]
```

Comment:
`[Exposed=AudioWorklet]` is the syntax that's actually used in specs, so I think this test can probably be removed together with the code that does the stripping.